### DISCUSSION TITLE
Copilot and tensorflow

### DISCUSSION THREAD

**Discussion by Alpusa** [community/community#13727] [Copilot Conversations]
I tried to make a test with Copilot to generate a basic neural network, but the tool only give me the imports of these libraries. I was waiting to get a basic structure but it was not successfully. I tried it using tensorflow and keras.

![image](https://user-images.githubusercontent.com/21032053/160507578-fbd4c783-dbf2-4ada-84b0-ad5c70aaac4c.png)

I'm using Visual Studio Code

**Comment by jayadeyemi** [comment]
I just experienced a similar thing. I have code already written but it's not even predicting comments, and now I'm searching for more information about why it fails to predict tensorflow code.
